src/mmw/js/src/compare/templates/compareWindow.html

```diff
@@ -1,8 +1,5 @@
 <div id="compare-scenarios-region"></div>
 <div id="compare-footer">
-    <div class="back"> <!-- Back -->
-      <a data-url="/project" class="btn btn-md btn-icon"><i class="fa fa-arrow-left"></i></a>
-    </div> <!-- End Back -->
     <h3>Navigate Scenarios</h3>
```

Comment:
I think it is referring to the arrows on the far right of the page. It is indeed confusing. I'll open an issue.